Added red notification (#1370)
spryker/spryker-docs: docs/scos/dev/back-end-development/extending-spryker/development-strategies/spryker-os-module-customisation/extend-a-core-module-that-is-used-by-another.md

{% endinfo_block %}

{% info_block errorBox %}

> Spryker is constantly improving type declararations of all methods, thus some bridge interfaces might be incompatible with Facade interfaces, so this approach will not work. To prevent this, please consider module version patch-lock.

Reviewer: ```suggestion
Spryker is constantly improving type declarations of all methods. Thus, some bridge interfaces might be incompatible with Facade interfaces, so this approach does not work. To prevent this, consider module version patch-lock.
```
